docs/source/getting-started/faq.rst

```diff
@@ -0,0 +1,106 @@
+
+**Q: How much technical background do we need to have?**
+
+A: Basic familiarity with Python and Jupyter notebooks is recommended.
+Knowledge of geospatial data handling is a plus but not mandatory for beginners.
+We have provided some `coding basics </in-depth-resources/coding-basics/>`_ in this documentation site to help you get started.
+
+**Q: What does the output of MapReader look like?**
+
+A: MapReader produces patch data, annotations, and model predictions.
+Outputs can include CSV files for metadata and labels, annotated patches in image formats, and trained models for further inference.
+
+**Q: How can I contribute to the codebase?**
+
+A: Contributions are welcome!
+You can fork the MapReader repository, make changes, and submit a pull request.
+Please review our `code contribution guide </community-and-contributions/contribution-guide/getting-started-with-contributions/add-or-update-code>`_ for details.
+
+**Q: Can I use MapReader to vectorize maps?**
+
+A: MapReader is not designed for full vectorization but can assist in creating training datasets for machine learning models, which can then infer vector-like features.
+
+**Q: Is it free to use MapReader?**
+
+A: Yes, MapReader is an open-source project and is free to use under its license.
+
+**Q: Why do I need to patchify my maps?**
+
+A: Patchifying maps is a crucial step in MapReader because it breaks down large map sheets into smaller, manageable regions called patches.
+Without the step of patchifying, processing large, complex maps would be resource-intensive and less flexible for targeted analysis or model training.
+Overall, this step is important as it enables some crucial features for you:
+
+- *Efficient data analysis*:
+  Analyzing smaller patches reduces computational overhead, making tasks like classification, annotation, and visualization more scalable.
+- *Better machine learning models*:
+  Most machine learning models process data in fixed-size inputs.
+  Patchifying maps ensures that the data conforms to the required dimensions for training and inference.
+- *Adaptability to the patch's context*:
+  Patchifying enables the use of context-based models, where information from surrounding patches can be incorporated for better predictions.
+
+**Q: How large do my patches need to be? (i.e., standards? best patch sizes?)**
+
+A: Patch sizes depend on the resolution of your maps and your analysis goals.
+Standard sizes are 100x100 or 300x300 pixels, but experimentation may be necessary for optimal results.
+You can read more in the `"Patchify" section </using-mapreader/step-by-step-guide/2-load.html#patchify>`_ in our step-by-step guide.
+
+**Q: What should I use for labels? What is a good label?**
+
+A: Labels should correspond to the features you want to identify in the maps.
+Common examples include "railspace," "urban," or "water."
+Custom labels can be tailored to your specific study area.
+
```

Comment:
Labels should correspond to a visual signal you know is present in the maps you are working with. This visual signal can have some variety, but too much variety will lead to poor results. Developing the name of a label and the guidelines for assigning a label to a patch is usually an iterative process that is refined as you start to annotate patches. Labels do not need to conform to typical map features, but they can. Previous examples of patch labels include "railspace", "building", "trees", "rocks", "water", "gardens", and "heath".